language-server/modules/langserver-core/src/test/resources/completion/new/statement_context/config/if_stmt_ctx_config6.json

```diff
@@ -1,470 +1 @@
on","insertTextFormat":"Snippet"},{"label":"typedesc","kind":"Unit","detail":"type","sortText":"Q","insertText":"typedesc","insertTextFormat":"Snippet"},{"label":"xml","kind":"Unit","detail":"type","sortText":"Q","insertText":"xml","insertTextFormat":"Snippet"},{"label":"service","kind":"Keyword","detail":"Keyword","sortText":"P","insertText":"service","insertTextFormat":"Snippet"},{"label":"new","kind":"Keyword","detail":"Keyword","sortText":"P","insertText":"new ","insertTextFormat":"Snippet"},{"label":"isolated","kind":"Keyword","detail":"Keyword","sortText":"P","insertText":"isolated ","insertTextFormat":"Snippet"},{"label":"transactional","kind":"Keyword","detail":"Keyword","sortText":"P","insertText":"transactional","insertTextFormat":"Snippet"},{"label":"function","kind":"Keyword","detail":"Keyword","sortText":"P","insertText":"function ","insertTextFormat":"Snippet"},{"label":"let","kind":"Keyword","detail":"Keyword","sortText":"P","insertText":"let","insertTextFormat":"Snippet"},{"label":"typeof","kind":"Keyword","detail":"Keyword","sortText":"P","insertText":"typeof ","insertTextFormat":"Snippet"},{"label":"trap","kind":"Keyword","detail":"Keyword","sortText":"P","insertText":"trap","insertTextFormat":"Snippet"},{"label":"error","kind":"Keyword","detail":"Keyword","sortText":"P","insertText":"error","insertTextFormat":"Snippet"},{"label":"client","kind":"Keyword","detail":"Keyword","sortText":"P","insertText":"client ","insertTextFormat":"Snippet"},{"label":"object","kind":"Keyword","detail":"Keyword","sortText":"P","insertText":"object ","insertTextFormat":"Snippet"},{"label":"true","kind":"Keyword","detail":"Keyword","sortText":"P","insertText":"true","insertTextFormat":"Snippet"},{"label":"false","kind":"Keyword","detail":"Keyword","sortText":"P","insertText":"false","insertTextFormat":"Snippet"},{"label":"check","kind":"Keyword","detail":"Keyword","sortText":"P","insertText":"check ","insertTextFormat":"Snippet"},{"label":"checkpanic","kind":"Keyword","detail":"Keyword","sortText":"P","insertText":"checkpanic ","insertTextFormat":"Snippet"},{"label":"is","kind":"Keyword","detail":"Keyword","sortText":"P","insertText":"is","insertTextFormat":"Snippet"},{"label":"error constructor","kind":"Snippet","detail":"Snippet","sortText":"O","insertText":"error(\"${1}\")","insertTextFormat":"Snippet"},{"label":"object constructor","kind":"Snippet","detail":"Snippet","sortText":"O","insertText":"object {${1}};","insertTextFormat":"Snippet"},{"label":"base16","kind":"Snippet","detail":"Snippet","sortText":"O","insertText":"base16 `${1}`","insertTextFormat":"Snippet"},{"label":"base64","kind":"Snippet","detail":"Snippet","sortText":"O","insertText":"base64 `${1}`","insertTextFormat":"Snippet"},{"label":"testFunction()","kind":"Function","detail":"Function","documentation":{"right":{"kind":"markdown","value":"**Package:** _._  \n  \n  \n  \n  \n**Returns** `()`   \n  \n"}},"sortText":"C","insertText":"testFunction()","insertTextFormat":"Snippet"}]}
```

Comment:
Seems the formatting is wrong in autogenerated LS assertion files. Check other places as well.

Reply:
Fixed.